jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -912,38 +913,224 @@ ecma_date_insert_num_with_sep (ecma_string_t **str_p, /**< input/output string *
   ecma_date_insert_leading_zeros (str_p, num, length);
 } /* ecma_date_insert_num_with_sep */
 
+
+/**
+ * Common function to copy utf8 characters.
+ *
+ * @return next destination buffer position
+ */
+
```

Comment:
I think there is an extra line here, usually there isn't space between the comment and the function.